plugins/modules/system/homectl.py

```diff
@@ -0,0 +1,650 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2022, James Livulpi
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: homectl
+author:
+    - "James Livulpi (@jameslivulpi)"
+short_description: Manage user accounts with systemd-homed
+version_added: 4.4.0
+description:
+    - Manages a user's home directory managed by systemd-homed.
+options:
+    name:
+        description:
+            - The user name to create, remove, or update.
+        required: true
+        aliases: [ 'user', 'username' ]
+        type: str
+    password:
+        description:
+            - Set the user's password to this.
+            - Homed requires this value to be in cleartext on user creation and updating a user.
+            - The module takes the password and generates a password hash in SHA-512 with 10000 rounds of salt generation using crypt.
+            - See U(https://systemd.io/USER_RECORD/).
+            - This is required for I(state=present). When an existing user is updated this is checked against the stored hash in homed.
+        type: str
+    state:
+        description:
+            - The operation to take on the user.
+        choices: [ 'absent', 'present' ]
+        default: present
+        type: str
+    storage:
+        description:
+            - Indicates the storage mechanism for the user's home directory.
+            - If the storage type is not specified, ``homed.conf(5)`` defines which default storage to use.
+            - Only used when a user is first created.
+        choices: [ 'classic', 'luks', 'directory', 'subvolume', 'fscrypt', 'cifs' ]
+        type: str
+    disksize:
+        description:
+            - The intended home directory disk space.
+            - Human readable value such as C(10G), C(10M), or C(10B).
+        type: str
+    resize:
+        description:
+            - When used with I(disksize) this will attempt to resize the home directory immediately.
+        default: false
+        type: bool
+    realname:
+        description:
+            - The user's real ('human') name.
+            - This can also be used to add a comment to maintain compatability from C(useradd).
```

Comment:
I think `with` makes more sense here (but then, I'm not a native English speaker, so it could also be I'm wrong):
```suggestion
            - This can also be used to add a comment to maintain compatability with C(useradd).
```

Reply:
Agreed !